feat: add logic to overwrite defaultsJson's build_branch and pipeline_branch and add helper_ref (#423)
adoptium/ci-jenkins-pipelines: docs/UsingOurScripts.md

        // Git Url of the current repository.
        "pipeline_url"       : "https://github.com/adoptium/ci-jenkins-pipelines.git",
        // Git branch you wish to use when running the groovy scripts inside the pipeline_url
        "pipeline_branch"    : "master",

>         // Git branch of which checkout from https://github.com/adoptium/jenkins-helper.git repo. This can only be as branch or tag, not SHA1

Reviewer: ```suggestion
        // Git branch name from the https://github.com/adoptium/jenkins-helper.git repo. This can only be as branch or tag, not SHA1
```
